pkg/api/callback.go

```diff
@@ -280,5 +305,16 @@ func UpdateRedemptionStatus(auth *util.AuthConfig,
 		return err
 	}
 
+	// After updating the status, we send a message to the chat
+	if success {
+		if err := chatbot.SendChatMessage(broadcaster, userStore, event); err != nil {
```

Comment:
If we get this far, not only do we already have an updated Spotify token to reference, we also already have the User struct above. We don't need to pass in the userStore here at all. The credentials in the `u` variable will be refreshed already. You could change this function call to send the chat message to take in the Spotify and Twitch tokens directly instead of doing an additional DB read. 

Separately, I think that my function was not written well enough because it passes in the userStore when I don't need to. I think this would be simpler if the `UpdateRedemptionStatus` function just accepted a `*User` struct. 

```go
func UpdateRedemptionStatus(auth *util.AuthConfig,
	user *users.User
	event *helix.EventSubChannelPointsCustomRewardRedemptionEvent,
	success bool) error {
```